src/javaRosa/util.js

```diff
@@ -270,8 +270,14 @@ define([
                 text = $("<div />").append(text);
                 text.find('output').replaceWith(function() {
                     var $this = $(this),
-                        value = xpathParser.parse($this.attr('value') || $this.attr('ref'));
-                    $this.attr('value', value[functionName]());
+                        value = $this.attr('value') || $this.attr('ref'),
+                        parsedValue;
+                    try {
+                        parsedValue = xpathParser.parse(value);
+                        $this.attr('value', parsedValue[functionName]());
+                    } catch (e) {
+                        $this.attr('value', value);
```

Comment:
I guess that means you can't use hashtags (like `#case/foo`) in typed output value expressions?